.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Release quarkus blueprint
```

Comment:
I've created a similar workflow in the past. I like to call it "Publish" for the name and filename. When the release is created, it's auto-published to npmjs.org. I'd also recommend switching to two spaces for YAML.

https://github.com/oktadev/schematics/blob/main/.github/workflows/publish.yml

Reply:
sure, will update